samples/dnn/classification.cpp

```diff
@@ -22,12 +22,11 @@ std::string keys =
     "{ classes          | | Optional path to a text file with names of classes. }"
     "{ backend          | 0 | Choose one of computation backends: "
                             "0: automatically (by default), "
-                            "1: Halide language (http://halide-lang.org/), "
-                            "2: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
-                            "3: OpenCV implementation, "
-                            "4: VKCOM, "
-                            "5: CUDA, "
-                            "6: WebNN }"
+                            "1: Intel's Deep Learning Inference Engine (https://software.intel.com/openvino-toolkit), "
+                            "2: OpenCV implementation, "
+                            "3: VKCOM, "
+                            "4: CUDA, "
+                            "5: WebNN }"
```

Comment:
Please use better approach to keep compatibility - use enum values such as `DNN_BACKEND_CUDA` and `cv::format`

Reply:
Done.